models/RoutesParser.cfc

```diff
@@ -463,6 +463,25 @@ component accessors="true" threadsafe singleton{
 					continue;
 				}
 
+				// security
+				if ( infoKey == "security" ) {
+					method.put( "security", [] );
+					if ( IsSimpleValue( infoMetadata ) ) {
+						// expect a list of pre-defined securitySchemes
```

Comment:
Hmm, so why are you inflating to an array using a space as a delimiter, I would have imagined it would be something like:

```
@security apikey,test,another
```
instead of 
```
@security apikey test another
```

Reply:
My recollection is that I was following a pattern from swagger, but I can't find it now. It would work either way